docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc

```diff
@@ -20,6 +21,26 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.1.2 relnotes
+
+[[release-notes-8.1.2]]
+== {fleet} and {agent} 8.1.2
+
+Review important information about the {fleet} and {agent} 8.1.2 release.
+
+[discrete]
+[[bug-fixes-8.1.2]]
+=== Bug fixes
+
+{fleet}::
+* There are no bug fixes for {fleet} in this release.
+
+{agent}::
+* Propagate input ID from the Agent policy into the Filebeat configuration. Note
+that no validation is performed on this field. {agent-pull}30386[#30386]
+
```

Comment:
* Fix the start sequence of Beats that was non-deterministic making Beats missing their 
configuration from Agent and not sending events. {agent-pull}30694[#30694]

Reply:
Adding this suggestion